ex1-shai/bootstrap.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+#Install docker
```

Comment:
See my comment about comments in other PRs

Reply:
I saw, 
Remove the comment?

Reply:
You should not get a permission from me to execute.